game/static/game/css/landing.css

```
@@ -146,20 +158,49 @@ body {
+    mask-image: radial-gradient(circle, black 0%, transparent 68%);
+    pointer-events: none;
+    z-index: -1;
 }
 
 .hero-title {
-    font-size: 2.8rem;
+    font-size: clamp(2.8rem, 7vw, 5.8rem);
     font-weight: 700;
     line-height: 1.25;
     margin-bottom: 1rem;
     color: var(--text-title);
```

Stylelint (17.11.0)

[error] Duplicate property "color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)